### PR TITLE
Section about events in MiniApp Components

### PR DESCRIPTION
**Pull request by espinr**
Section with the list of common events in MiniApps components (+@MichaelWangzitao)

***
<a href="https://pr-preview.s3.amazonaws.com/espinr/miniapp-components/pull/13.html" title="Last updated on May 31, 2023, 10:57 AM UTC (89f1167)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-components/13/39b4830...espinr:89f1167.html" title="Last updated on May 31, 2023, 10:57 AM UTC (89f1167)">Diff</a>

### CODE REVIEW

**Comment by espinr** [comment]
I've merged all the branches (one per individual section) into one to have a full picture of the document: https://raw.githack.com/espinr/miniapp-components/all-new-sections/index.html#sec-events

File: index.html [hunk 1]
```diff
@@ -1,1642 +1,264 @@
...
+        <dt>`click` or `tap`</dt>
+        <dd>Like the <a href="https://www.w3.org/TR/uievents/#event-type-click" data-link-type="dfn">click</a></th> event type defined in [[UIEVENTS]], this event is dispatched on the topmost event target indicated by the pointer when the user presses down and releases the primary pointer button or otherwise activates the pointer in a manner that simulates such an action.</dd>
+        <dt>`longtap` or `longpress`</dt>
+        <dd>The `longpress` or `longtap` event type is dispatched on the topmost component (event target) indicated by the pointer when the user presses down, holds for at least (<mark>@@@ms@@@</mark>), and releases the primary pointer button or otherwise activates the pointer in a manner that simulates such an action.</dd>
+      </dl>
+      <div class="ednote">
+        Could we define the minimum period to be considered as a long press?
```
Comment (Sharonzd) [inline index.html]: I think we could define the minimum period.  
As a developer, I want to know how long it is.
If possible, maybe we can provide a configuration that allows developers to modify the default duration in the future.

**Review by Sharonzd — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (espinr) [reply index.html]: I haven't found the durations in the specific implementations. 
Can we say that can it MUST configured by the platform by default and MAY be modified by developers?

Comment (espinr) [reply index.html]: I've added a note with the value that Android uses for this event. As reference, but not mandatory. Please have a look: 89f116779ab7cdd286a1a846e1786b03234d0615

**Pull request merged**